Helm2; release v0.13.2 (#511)
giantswarm/chart-operator: CHANGELOG.md

## [Unreleased]

## [v0.13.2] 2020-06-23

### Changed

> - Metrics for the helm release which contains the different namespace.

Reviewer: ```suggestion
- Add metrics for Helm releases with a mismatched namespace.
```

## [v0.13.1] 2020-06-04

### Changed

> - Calculating md5sum from `Chart` go struct.

Reviewer: <3
